improver/ensemble_copula_coupling/ensemble_copula_coupling.py

```diff
@@ -993,6 +1013,11 @@ def _location_and_scale_parameters_to_probabilities(
                 probability_method(threshold), probabilities.shape[1:])
 
         probability_cube = probability_cube_template.copy(data=probabilities)
+        # Make the mask defined above fit the data size and then apply to the
+        # probability cube.
+        mask_array = np.array([mask]*len(probabilities))
+        probability_cube.data = np.ma.masked_where(
+            mask_array is True, probability_cube.data)
```

Comment:
Is there a reason for the `is True` here?

Reply:
I forgot to change it when make the other changes to simplify the code. I have updated this now.